src/pages/GetStarted/GetStartedResults.vue

```diff
@@ -1,464 +0,0 @@
-					<div class="how-it-works__circle"></div>
-					<div class="how-it-works__circle"></div>
-				</div>
-				<ol class="how-it-works__list row align-justify">
-					<li class="how-it-works__li small-12 large-4 columns">
-						<kv-responsive-image
-							class="how-it-works__img how-it-works__img--loan show-for-large"
-							:images="howItWorksImgs.loan"
-							loading="lazy"
-							alt=""
-						/>
-						<h3 class="tw-mb-1 md:tw-mb-2">
-							Support a loan<br class="xxlu"> today!
-						</h3>
-						<p>
-							Make an impact with as<br class="xxlu"> little as $25.
-						</p>
-					</li>
-					<li class="how-it-works__li small-12 large-4 columns">
-						<kv-responsive-image
-							class="how-it-works__img how-it-works__img--people show-for-large"
-							:images="howItWorksImgs.people"
-							loading="lazy"
-							alt=""
-						/>
-						<h3 class="tw-mb-1 md:tw-mb-2">
-							Lenders around the world<br class="xxlu"> help fund the loan
-						</h3>
-						<p>This takes up to thirty days.</p>
-					</li>
-					<li class="how-it-works__li small-12 large-4 columns">
-						<kv-responsive-image
-							class="how-it-works__img how-it-works__img--repaid show-for-large"
-							:images="howItWorksImgs.repaid"
-							loading="lazy"
-							alt=""
-						/>
-						<h3 class="tw-mb-1 md:tw-mb-2">
-							Get repaid after your<br class="xxlu"> borrower uses their loan
-						</h3>
-						<p>You can use the repayments to support more borrowers or withdraw it.</p>
-					</li>
-				</ol>
-			</section>
-
-			<section class="section">
-				<frequently-asked-questions />
-				<kv-responsive-image
-					class="faq__flourish"
-					:images="flourishImgs.greenLeft"
-					loading="lazy"
-					alt=""
-				/>
-			</section>
-		</div>
-	</div>
-</template>
-
-<script>
-import { gql } from 'graphql-tag';
-import * as Sentry from '@sentry/vue';
-import FrequentlyAskedQuestions from '#src/components/GetStarted/FrequentlyAskedQuestions';
-import EditPreferences from '#src/components/GetStarted/EditPreferences';
-import KvProgressBar from '#src/components/Kv/KvProgressBar';
-import KvResponsiveImage from '#src/components/Kv/KvResponsiveImage';
-import RecommendedLoanCard from '#src/components/LoanCards/RecommendedLoanCard';
-import logFormatter from '#src/util/logFormatter';
-import { metaGlobReader } from '#src/util/importHelpers';
-
-const imgGlob = import.meta.glob('/src/assets/images/lend-by-category-homepage/*.*', { eager: true });
-const imgRequire = metaGlobReader(imgGlob, '/src/assets/images/lend-by-category-homepage/');
-
-export default {
-	name: 'GetStartedResults',
-	inject: ['apollo', 'cookieStore'],
-	components: {
-		FrequentlyAskedQuestions,
-		EditPreferences,
-		KvProgressBar,
-		KvResponsiveImage,
-		RecommendedLoanCard,
-	},
-	data() {
-		return {
-			loanIds: [],
-			totalPreferredCount: 0,
-			totalFundRaisingCount: 0,
-			howItWorksImgs: {
-				loan: [
-					['small', imgRequire('how-it-works-loan.png')],
```

Comment:
Looks like these images are no longer needed, remove?

Reply:
@tiarascoleman yeah any import in these files that aren't used elsewhere can be removed.